src/test/scala/org/clulab/habitus/variables/TestVariableReaderLabelBased.scala

```diff
@@ -348,7 +342,8 @@ class TestVariableReaderLabelBased extends Test {
       "sent21_13", "Harvests have started in some production areas of the valley, to date an area estimated at 843 ha is already harvested in  the Delta, 199 ha in Matam, 31 ha in Bakel, and 23 ha in Dagana.",
       "PlantingArea",
       Seq(("Area", Seq(("843 ha", "843.0 ha"), ("199 ha", "199.0 ha"), ("31 ha", "31.0 ha"), ("23 ha", "23.0 ha"))))
-    )
+    ),
```

Comment:
We are only compiling for Scala 2.12, but if we were also doing 2.11, then this dangling comma would cause a problem.  No need to change, but FYI.

Reply:
Okay, well noted. 

Thank you, @kwalcock